### PR TITLE
Fix wazuh_kibana package url and update Kibana certificates

### PR DESCRIPTION
**Pull request by BraulioV**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-jenkins/issues/1967|

## Description

Hi team,

This PR closes https://github.com/wazuh/wazuh-jenkins/issues/1967 by fixing the URL selector for the `wazuh-kibana` package and it also updates the Kibana certificates used to set up https access.
## Tests

- Build the package in any supported platform
  - [ ] OVA

### CODE REVIEW

**Review by Rebits — APPROVED** [review]
LGTM

File: ova/Libraries/provision-opendistro.sh [hunk 1]
```diff
@@ -236,7 +236,7 @@ installKibana() {
         fi
         mkdir -p /etc/kibana/certs
         cp /etc/elasticsearch/certs/root-ca.pem /etc/kibana/certs/
-        mv /etc/elasticsearch/certs/{kibana.key,kibana.pem} /etc/kibana/certs/
+        mv /etc/elasticsearch/certs/{kibana_http.key,kibana_http.pem} /etc/kibana/certs/
```
Comment (DFolchA) [inline ova/Libraries/provision-opendistro.sh]: The `kibana.yml` file that is downloaded in this line:

https://github.com/wazuh/wazuh-packages/blob/c518fb2d2184516e17dbd4bc410524aab39d1efa/ova/Libraries/provision-opendistro.sh#L218

References the certificate files as `kibana.key` and `kibana.pem`

https://github.com/wazuh/wazuh-documentation/blob/0dbb994f2c9c8f60d7d1b36330d65ed349d4c5c4/resources/open-distro/kibana/7.x/kibana_all_in_one.yml#L11

We need to rename them or change the `kibana.yml` file.

**Review by DFolchA — CHANGES_REQUESTED** [review]

File: ova/Libraries/provision-opendistro.sh (hunk 1 above)
Comment (BraulioV) [reply ova/Libraries/provision-opendistro.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/534/commits/6166e85603757c16f5ae469be9373dcdecf6751a

**Review by DFolchA — APPROVED** [review]
LGTM

**Pull request merged**